db/migrate/20170317142609_create_searches.blacklight.rb

```diff
@@ -1,6 +1,6 @@
 # This migration comes from blacklight (originally 20140202020201)
 # frozen_string_literal: true
-class CreateSearches < ActiveRecord::Migration
+class CreateSearches < ActiveRecord::Migration[4.2]
```

Comment:
Maybe you know more about this, but from what I can find the number here is the rails version target? Should these be `5.2` or did something tell you `4.2` is right?

Reply:
I tried 5.2 but there are other issues that arise from that. All the other migrations in the app emulate the 4.2 change, so its actually more consistent to have it this way specifically for hyrax.